<?xml version="1.0"?>
<robot name="bucket">
  <link name="base_link">
    <visual>
      <origin xyz="0 0 0" rpy="0 0 -0.785398163" />
      <geometry>
  <mesh filename="package://hri_scenario/meshes/bucket.dae" scale="0.05 0.05 0.05"/>
      </geometry>
    </visual>
  </link>
</robot>
